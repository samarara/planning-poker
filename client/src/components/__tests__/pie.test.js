// import { shallow } from "enzyme";
import TestRenderer from 'react-test-renderer'; // ES6
import Pie from "../pie";
// import React from "react"

describe("pie chart", () => {
  it("should match snapshot", () => {
    // const pie = shallow(<Pie />);
    const pie = TestRenderer.create(<Pie />)
    expect(pie).toMatchSnapshot();
  });
});
