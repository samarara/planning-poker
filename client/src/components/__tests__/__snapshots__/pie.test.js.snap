// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pie chart should match snapshot 1`] = `
<div
  className="MuiContainer-root makeStyles-pie-1 MuiContainer-maxWidthLg"
>
  <div
    style={
      Object {
        "height": "100%",
        "width": "100%",
      }
    }
  />
</div>
`;
